src/Conversion/ONNXToKrnl/Tensor/Concat.cpp

```diff
@@ -50,29 +52,31 @@ struct ONNXConcatOpLowering : public ConversionPattern {
       for (int r = 0; r < rank; ++r)
         inputLoops.pushBounds(0, operands[i], r);
       inputLoops.createIterateOp();
+      // writeOffset = writeOffset +
```

Comment:
If this is no longer needed, remove please

Reply:
deleted